src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -278,13 +459,19 @@ ucs_status_t uct_rc_mlx5_iface_common_init(uct_rc_mlx5_iface_common_t *iface,
        rc_iface->config.atomic64_ext_handler = uct_rc_mlx5_common_atomic64_le_handler;
     }
 
+complete:
```

Comment:
this should be "return UCS_OK" and line 467 should "return status"

Reply:
fixed